subworkflows/local/map.nf

```diff
@@ -2,16 +2,16 @@
 // Prepare reference indexing for downstream
 //
 
-include { FASTQ_ALIGN_BWAALN } from '../../subworkflows/nf-core/fastq_align_bwaaln/main'
-include { SAMTOOLS_FLAGSTAT  } from '../../modules/nf-core/samtools/flagstat/main'
+include { FASTQ_ALIGN_BWAALN                             } from '../../subworkflows/nf-core/fastq_align_bwaaln/main'
+include { SAMTOOLS_FLAGSTAT as SAMTOOLS_FLAGSTAT_MAPPED  } from '../../modules/nf-core/samtools/flagstat/main'
```

Comment:
ok. this notation is better. maybe we could rename the modules in deduplicate.nf to follow this system. easier to work with the modules.config too 🤔

Reply:
not sure what you're referring to here, but OK...?